src/main/java/graphql/language/ObjectValue.java

```diff
@@ -77,7 +81,8 @@ public boolean isEqualTo(Node o) {
 
     @Override
     public ObjectValue deepCopy() {
-        return new ObjectValue(deepCopy(objectFields), getSourceLocation(), getComments(), getIgnoredChars(), getAdditionalData());
+        List<ObjectField> copiedFields = deepCopy(objectFields);
+        return new ObjectValue(copiedFields != null ? copiedFields : emptyList(), getSourceLocation(), getComments(), getIgnoredChars(), getAdditionalData());
```

Comment:
Similar to ArrayValue, want to call out this is a small change to not allow a null value. A list will be used instead. Without this change, there is a NullAway error